mailet/standard/src/test/java/org/apache/james/transport/matchers/HasMimeTypeTest.java

```diff
@@ -0,0 +1,60 @@
+package org.apache.james.transport.matchers;
+
+import org.apache.james.core.MailAddress;
+import org.apache.mailet.base.test.FakeMail;
+import org.apache.mailet.base.test.FakeMatcherConfig;
+import org.apache.mailet.base.test.MimeMessageBuilder;
+import org.junit.Before;
+import org.junit.Test;
+
+import javax.mail.internet.MimeMessage;
+
+import static org.junit.Assert.assertNotNull;
+
+public class HasMimeTypeTest {
+
+    private HasMimeType matcher;
+    private final String RECIPIENT = "test@james.apache.org";
+    private final String FROM = "test@james.apache.org";
+    private final String MIME_TYPES = "multipart/mixed";
+    private MimeMessage message;
+    private FakeMail mail;
+
+    @Before
+    public void setUp() throws Exception {
+        matcher = new HasMimeType();
+        message = MimeMessageBuilder.mimeMessageBuilder()
+                .setMultipartWithBodyParts(
+                        MimeMessageBuilder.bodyPartBuilder()
+                                .data("simple text")
+                                .disposition("text")
+                                .build(),
+                        MimeMessageBuilder.bodyPartBuilder()
+                                .filename("text_file.txt")
+                                .disposition("attachment")
+                                .build(),
+                        MimeMessageBuilder.bodyPartBuilder()
+                                .type("application/zip")
+                                .filename("zip_file.zip")
+                                .disposition("attachment")
+                                .build())
+                .setSubject("test")
+                .build();
+
+        mail = FakeMail.builder()
+                .mimeMessage(message)
+                .sender(new MailAddress(FROM))
+                .recipient(new MailAddress(RECIPIENT))
+                .build();
+    }
+
+    @Test
+    public void hasMimeType() throws Exception {
+        matcher.init(FakeMatcherConfig.builder()
+                .matcherName("HasMimeType")
+                .condition(MIME_TYPES)
+                .build());
+        assertNotNull(matcher.match(mail));
```

Comment:
Can we be more explicit than is not null? It sould contain the list of recipients.